examples/fargate/main.tf

```diff
@@ -90,15 +90,25 @@ module "eks" {
   vpc_id = module.vpc.vpc_id
 
   fargate_profiles = {
-    example = {
-      namespace = "default"
-
-      # Kubernetes labels for selection
-      # labels = {
-      #   Environment = "test"
-      #   GithubRepo  = "terraform-aws-eks"
-      #   GithubOrg   = "terraform-aws-modules"
-      # }
+    default = {
+      name = "default"
+      selectors = [
+        {
+          namespace = "kube-system"
+          labels = {
+            "k8s-app" : "kube-dns"
+          }
+        },
```

Comment:
Can you please update the doc https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/modules/fargate/README.md#fargate_profile-keys accordingly

And `k8s-app =  "kube-dns"` instead of `"k8s-app" :  "kube-dns"`